lib/dialects/sqlite/data-types.js

```diff
@@ -97,11 +97,41 @@ FLOAT.prototype.toSql = function() {
   return NUMBER.prototype.toSql.call(this);
 };
 
+var DOUBLE = function(length, decimals) {
```

Comment:
I can't tell what's different here for these two types compared to the abstract one.